pyredis.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import socket
+
+HOST = "127.0.0.1"  # Local host IP
+PORT = 6379  # Current port used by the Redis server
+SPACING = "\r\n"
+ARRAY_START = "*"
+BULK_STR_START = "$"
+
+
+def main():
+    # Start argparse for accepting commands from client
+    parser = argparse.ArgumentParser(
+        description="This will print a statement passed as argument",
+        epilog="Why are you even reading the help page for this?",
+    )
+
+    # required argument for 3 commands [GET, SET, DEL] and its proceeding argument providing keys to be retrieved, set or deleted.
+    parser.add_argument(
+        "message",
+        type=str,
+        choices=["GET", "SET", "DEL"],
+        default="You didn't input anything",
+    )
+    parser.add_argument("key_or_value", type=str, nargs="+")
+
+    # Potential future command when working with the need to change ports:
+
+    # parser.add_argument('--port','-p',type=int,default=6379, help='This is the default port to connect to')
+
+    # Generates redis formatted array from CLI input.
+    args = parser.parse_args()
```

Comment:
this is more of a style comment, so take it or leave it

I generally think of `main` as doing the work of the program, so I tend to put the bulk of the logic in `main`. Personally I would put argument parsing in to its own helper function and un-helper-function the communication with the redis server.